### PR TITLE
add online address in README file and link on the scroll icon

### PR DESCRIPTION
**Pull request by Grandi0z**
### In this version we :
- add the address of the online version 
- add a link to the scroll button in the desktop version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ You can visite the online version at : <a href="https://grandi0z.github.io/#">gr
 👤 **Author1**
 
 - GitHub: [@Grandi0z](https://github.com/Grandi0z)
-- Twitter: [@wa_inje](https://twitter.com/wa_inje)
+- Twitter: [@wa_inje](https://twitter.com/Joseph_LSK)
```
Comment (andgarzonmal) [inline README.md]: - [ ] Hi👋, nice job updating your social media links, but please remember you also need to add to your readme file the link to your deployed page using GitHub pages.

Comment (andgarzonmal) [inline README.md]: - [ ]  Please check the online version of your portfolio, and make sure that the page works fine. ✔️

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @   ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by levy002 — APPROVED** [review]
## Hello @Grandi0z 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to add the link of your deployed portfolio to the `LIVE DEMO` section in your README file.
 
   image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Grandi0z) [reply README.md]: Hi,  @andgarzonmal. Thank you for your review. I've just made another request review. I think I will check if the online version works perfectly  when I am allowed to marge changes

**Pull request merged**